core-feature-pack/src/main/resources/content/bin/domain.sh

```diff
@@ -96,6 +98,12 @@ if [ "x$JAVA" = "x" ]; then
     fi
 fi
 
+# Set default modular JVM options
+setDefaultModularJvmOptions $PROCESS_CONTROLLER_JAVA_OPTS
+PROCESS_CONTROLLER_JAVA_OPTS="$PROCESS_CONTROLLER_JAVA_OPTS $DEFAULT_MODULAR_JVM_OPTIONS"
+setDefaultModularJvmOptions $HOST_CONTROLLER_JAVA_OPTS
+HOST_CONTROLLER_JAVA_OPTS="$HOST_CONTROLLER_JAVA_OPTS $DEFAULT_MODULAR_JVM_OPTIONS"
```

Comment:
This needs to be tested, but it may need to be invoke later. Essentially once all the JAVA_OPTS are filled.